pytest_enabler/__init__.py

```diff
@@ -99,9 +129,13 @@ def _pytest_cov_check(plugins, early_config, parser, args):  # pragma: nocover
     _remove_deps()
     # important: parse all known args to ensure pytest-cov can configure
     # itself based on other plugins like pytest-xdist (see #1).
+    if parser is None:
+        raise ValueError("parser cannot be None if cov in plugins")
     parser.parse_known_and_unknown_args(args, early_config.known_args_namespace)
 
-    with contextlib.suppress(ImportError):
+    try:
         import pytest_cov.plugin
+    except ImportError:
+        pass
```

Comment:
I prefer the more concise form. `mypy` shouldn't be enforcing gratuitous blocks.

Reply:
Good to know. I agree it might be worthwhile to performance-optimize, just not prematurely and not here, as you say.